### ISSUE TITLE
Feature request: UUID validator

### ISSUE THREAD

**Issue by yaronsumel** [mwitkow/go-proto-validators#40]
Hey @mwitkow ! 
Thanks for this great project. Would it be possible to include useful validators such as UUID  ? (ie. not in a fork version for this project.. I'll create the PR for it).. At the moment we need to copy and paste it all over the protos.

```proto
string user_id = 1 [(validator.field) = {regex: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"}];
```

Proposal usage:
```proto
string user_id = 1 [(validator.field) = {uuid : true}];
```

**Comment by rogchap** [comment]
Would be great to support `bytes` as well as string for as UUID eg:
```
bytes user_id = 1 [(validator.field) = {uuid : true}];
```

**Comment by Helcaraxan** [comment]
Closed via #70.
